packages/block-library/src/query-pagination-numbers/index.php

```diff
@@ -38,12 +38,14 @@ function render_block_core_query_pagination_numbers( $attributes, $content, $blo
 		$prev_wp_query = $wp_query;
 		$wp_query      = $block_query;
 		$total         = ! $max_page || $max_page > $wp_query->max_num_pages ? $wp_query->max_num_pages : $max_page;
+		$mid_size      = $block->attributes['midSize'] ? (int) $block->attributes['midSize'] : null;
```

Comment:
Haven't tested yet, but it seems if we have set `0` it just works due to the `(int)` coercion of `null`, that also is `0`. We should probably check if `isset` or something like that.

Additionally we would want the same arg in the case the `query` inherits the global query.

Reply:
`mid_size` is used for `paginate_links` should also be passed in the case we have this block inside a `Query Loop` block that inherits the global query. So we would need to pass this argument in this line: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/query-pagination-numbers/index.php#L33. 

You can test that by using a block theme and go for example in `archive` template and test Query Loop block there with the `mid_size` attribute.

Reply:
Alright, I'll give that a try. Do you agree with me that in this case the midSize control for the parameter should be hidden in the editor and the global_query value be used (if that is even accessible from the backend)?

Reply:
Okay, now I understood what you meant. I now pass the argument in both cases: Inherited query and not inherited. I also make sure, that the parameter is set at all. Otherwise it will not be sent as an argument.